ignite/templates/typed/list/list.go

```diff
@@ -16,34 +16,34 @@ import (
 )
 
 var (
-	//go:embed stargate/component/* stargate/component/**/*
-	fsStargateComponent embed.FS
+	//go:embed files/component/* files/component/**/*
+	fsComponent embed.FS
 
-	//go:embed stargate/messages/* stargate/messages/**/*
-	fsStargateMessages embed.FS
+	//go:embed files/messages/* files/messages/**/*
+	fsMessages embed.FS
 
-	//go:embed stargate/simapp/* stargate/simapp/**/*
-	fsStargateSimapp embed.FS
+	//go:embed files/simapp/* files/simapp/**/*
+	fsSimapp embed.FS
 )
 
-// NewStargate returns the generator to scaffold a new type in a Stargate module
-func NewStargate(replacer placeholder.Replacer, opts *typed.Options) (*genny.Generator, error) {
+// NewGenerator returns the generator to scaffold a new type in a module
+func NewGenerator(replacer placeholder.Replacer, opts *typed.Options) (*genny.Generator, error) {
 	var (
 		g = genny.New()
 
 		messagesTemplate = xgenny.NewEmbedWalker(
-			fsStargateMessages,
-			"stargate/messages/",
+			fsComponent,
```

Comment:
```suggestion
			fsMessages,
```

Reply:
Despite this fix, there's still something wrong with `ignite s list`. Here is what I got : 

```sh
$ ignite s list mylist name --module toto --clear-cache
Error while running command /tmp/protoc1137517531 -I /tmp/1452564228 -I /home/tom/src/ignite/removestargate/proto -I /home/tom/go/pkg/mod/github.com/cosmos/ibc-go/v5@v5.0.1/proto -I /home/tom/go/pkg/mod/github.com/cosmos/ibc-go/v5@v5.0.1/third_party/proto -I /home/tom/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.46.4/proto -I /home/tom/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.46.4/third_party/proto --gocosmos_out=plugins=interfacetype+grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. /home/tom/src/ignite/removestargate/proto/removestargate/toto/genesis.proto /home/tom/src/ignite/removestargate/proto/removestargate/toto/mytype.proto /home/tom/src/ignite/removestargate/proto/removestargate/toto/packet.proto /home/tom/src/ignite/removestargate/proto/removestargate/toto/params.proto /home/tom/src/ignite/removestargate/proto/removestargate/toto/query.proto /home/tom/src/ignite/removestargate/proto/removestargate/toto/tx.proto: removestargate/toto/mylist.proto: File not found.
removestargate/toto/genesis.proto:6:1: Import "removestargate/toto/mylist.proto" was not found or had errors.
removestargate/toto/genesis.proto:15:12: "Mylist" is not defined.
: exit status 1
```

The content of the repo after this failure : 

```sh
$ git st
## master
 M go.mod
 M go.sum
 M proto/removestargate/toto/genesis.proto
 M proto/removestargate/toto/query.proto
 M proto/removestargate/toto/tx.proto
 M x/toto/client/cli/query.go
 M x/toto/client/cli/tx.go
 M x/toto/genesis.go
 M x/toto/genesis_test.go
 M x/toto/module_simulation.go
 M x/toto/types/codec.go
 M x/toto/types/genesis.go
 M x/toto/types/genesis_test.go
 M x/toto/types/keys.go
?? files/
?? x/toto/client/cli/tx_mylist.go
?? x/toto/client/cli/tx_mylist_test.go
?? x/toto/keeper/msg_server_mylist.go
?? x/toto/keeper/msg_server_mylist_test.go
?? x/toto/simulation/mylist.go
?? x/toto/types/messages_mylist.go
?? x/toto/types/messages_mylist_test.go
```

Reply:
Looks like `files` folder is not trimmed properly : 

```sh
$ tree files
files
└── messages
    └── x
        └── toto
            ├── client
            │   └── cli
            │       ├── tx_mylist.go
            │       └── tx_mylist_test.go
            ├── keeper
            │   ├── msg_server_mylist.go
            │   └── msg_server_mylist_test.go
            └── types
                ├── messages_mylist.go
                └── messages_mylist_test.go

7 directories, 6 files
```